build/pluginctl/main.go

```diff
@@ -70,6 +67,8 @@ func getClient() (*model.Client4, error) {
 		return client, nil
 	}
 
+	log.Printf("No socket found at %s for local mode deployment. Attempting to authenticate with credentials.", socketPath)
```

Comment:
nit: this is entirely redundant if I intentionally set the other environment variables. Can we only emit this if `MM_LOCALSOCKETPATH` was non-empty?

Reply:
Done!